src/api/axiosInstance.ts

```
@@ -78,34 +77,22 @@ async function refreshAccessToken(): Promise<string | null> {
-          detail: { accessToken: newAccess ?? null, refreshToken: newRefresh ?? null },
-        })
-      );
-    } catch {}
-
-    // 구독자들에게 새 토큰 전달
-    notifyTokenRefreshed(newAccess ?? null);
 
+    window.dispatchEvent(new CustomEvent('auth:tokenRefreshed', { detail: { accessToken: newAccess, refreshToken: newRefresh } }));
+    notifyTokenRefreshed(newAccess ?? null);
     return newAccess ?? null;
   } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)